@yonigozlan just ran your script and got the following output:
```
tests/models/qwen2_vl/test_video_processing_qwen2_vl.py: FAILED                                                                   [100%]
tests/models/qwen2_vl/test_video_processing_qwen2_vl.py: FAILED tests/models/qwen2_vl/test_video_processing_qwen2_vl.py::Qwen2VLVideoProcessingTest::test_can_compile_fast_video_processor - AssertionError: Tensor-likes are not close!
tests/models/bridgetower/test_image_processing_bridgetower.py: tests/models/bridgetower/test_image_processing_bridgetower.py::BridgeTowerImageProcessingTest::test_can_compile_fast_image_processor FAILED [100%]
tests/models/bridgetower/test_image_processing_bridgetower.py: FAILED tests/models/bridgetower/test_image_processing_bridgetower.py::BridgeTowerImageProcessingTest::test_can_compile_fast_image_processor - AssertionError: Tensor-likes are not close!
tests/models/perceiver/test_image_processing_perceiver.py: tests/models/perceiver/test_image_processing_perceiver.py::PerceiverImageProcessingTest::test_can_compile_fast_image_processor FAILED [100%]
tests/models/perceiver/test_image_processing_perceiver.py: FAILED tests/models/perceiver/test_image_processing_perceiver.py::PerceiverImageProcessingTest::test_can_compile_fast_image_processor - AssertionError: Tensor-likes are not close!
tests/models/llava_next/test_image_processing_llava_next.py: tests/models/llava_next/test_image_processing_llava_next.py::LlavaNextImageProcessingTest::test_can_compile_fast_image_processor FAILED [100%]
tests/models/llava_next/test_image_processing_llava_next.py: FAILED tests/models/llava_next/test_image_processing_llava_next.py::LlavaNextImageProcessingTest::test_can_compile_fast_image_processor - AssertionError: Tensor-likes are not close!
```

Although there are failures, output was the same without the changes in this branch. Wdyt?

--- a/src/transformers/models/bridgetower/image_processing_bridgetower_fast.py
+++ b/src/transformers/models/bridgetower/image_processing_bridgetower_fast.py
@@ -164,13 +164,18 @@ class BridgeTowerImageProcessorFast(BaseImageProcessorFast):
             raise ValueError(f"The `size` dictionary must contain the key `shortest_edge`. Got {size.keys()}")
         shorter = size.shortest_edge
         longer = int(1333 / 800 * shorter)
-        output_size = get_resize_output_image_size(
+        output_height, output_width = get_resize_output_image_size(
             image,
             shorter=shorter,
             longer=longer,
             size_divisor=size_divisor,
         )
-        return F.resize(image, output_size, interpolation=interpolation, antialias=antialias)
+        return super().resize(
+            image=image,
+            size=SizeDict(height=output_height, width=output_width),
+            interpolation=interpolation,
+            antialias=antialias,
+        )
 
     def center_crop(
         self,

--- a/src/transformers/models/llava_next/image_processing_llava_next_fast.py
+++ b/src/transformers/models/llava_next/image_processing_llava_next_fast.py
@@ -137,7 +137,11 @@ class LlavaNextImageProcessorFast(BaseImageProcessorFast):
         new_height, new_width = get_patch_output_size(image, target_resolution, input_data_format)
 
         # Resize the image
-        resized_image = F.resize(image, (new_height, new_width), interpolation=interpolation)
+        resized_image = self.resize(
+            image=image,
+            size=SizeDict(height=new_height, width=new_width),
+            interpolation=interpolation,
+        )
 
         return resized_image
 

--- a/src/transformers/utils/import_utils.py
+++ b/src/transformers/utils/import_utils.py
@@ -481,7 +481,7 @@ def is_cuda_platform():
     if is_torch_available():
         import torch
 
-        torch.version.cuda is not None
+        return torch.version.cuda is not None
     else:
         return False
 
@@ -490,7 +490,7 @@ def is_rocm_platform():
     if is_torch_available():
         import torch
 
-        torch.version.hip is not None
+        return torch.version.hip is not None
     else:
         return False
 

